Makefile

```diff
@@ -0,0 +1,9 @@
+PROTO_ROOT := bridge/sdk-core/protos/api_upstream/
```

Comment:
Can we avoid a Makefile? It's platform specific. Does Ruby have a non-platform-specific approach here such as using that Rakefile?